packages/pact-config/pact/pacts/genericmenuconsumer-genericapi.json

```diff
@@ -0,0 +1,43 @@
+{
```

Comment:
This should get overriden by the tests. They are setting the env variables in scope.